google-cloud-testing/spanner-benchwrapper/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>spanner-benchwrapper</artifactId>
+  <version>0.0.0</version>
+  <packaging>jar</packaging>
+  <groupId>com.google.cloud</groupId>
+
+  <properties>
+    <grpcVersion>1.24.1</grpcVersion>
```

Comment:
This is a little out of date. Could we use libraries-bom:3.0.0 to set versions?

Reply:
Hi @elharo, the latest `grpc-bom` is 1.25.0. Here the code depends on `com.google.api.grpc:grpc-google-cloud-spanner-v1:jar:1.46.1-SNAPSHOT`, which has a dependency on `grpc-protobuf:1.24.1`. 

```bash
$mvn dependency:tree
...
[INFO] |  +- com.google.api.grpc:grpc-google-cloud-spanner-v1:jar:1.46.1-SNAPSHOT:compile
[INFO] |  |  +- io.grpc:grpc-stub:jar:1.24.1:compile
[INFO] |  |  \- io.grpc:grpc-protobuf:jar:1.24.1:compile
[INFO] |  |     \- io.grpc:grpc-protobuf-lite:jar:1.24.1:compile
...
```

Could you please explain a bit more? Thanks.

Reply:
see https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google-Cloud-Platform-Libraries-BOM

Reply:
Thanks for the doc. I can add `libraries-bom:3.0.0` into `dependencyManagement`. But I wonder here `grpcVersion` is only used to configure `pluginArtifact` but not used for specifying the version, is it necessary to add `libraries-bom:3.0.0`? Because the pom file does not explicitly import `grpc-protobuf` as a dependency.

Reply:
I think I got what you mean here. I will update it. But I still need to specify the version for `pluginArtifact` and `protocArtifact`.